src/xb-opcode.c

```diff
@@ -257,6 +268,22 @@ xb_opcode_get_str (XbOpcode *self)
 	return self->ptr;
 }
 
+/**
+ * xb_opcode_get_tokens:
+ * @self: a #XbOpcode
+ *
+ * Gets the tokenized string stored on the opcode.
+ *
+ * Returns: a #GStrv, or %NULL if unset
```

Comment:
Add `(nullable)`

Reply:
I don't think it is -- it's a static array of size `XB_OPCODE_TOKEN_MAX+1` i.e. always allocated.

Reply:
Ooops, I got mixed up with the `token_idxs` array, which is lazily allocated in `xb-builder-node.c`. Sorry!

Reply:
But the “or `%NULL` if unset” does need dropping in that case

Reply:
Agree, fixed.